packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -917,6 +936,9 @@ mixin DirectionalFocusTraversalPolicyMixin on FocusTraversalPolicy {
 ///  * [OrderedTraversalPolicy], a policy that describes the order
 ///    explicitly using [FocusTraversalOrder] widgets.
 class WidgetOrderTraversalPolicy extends FocusTraversalPolicy with DirectionalFocusTraversalPolicyMixin {
+  /// Constructs a traversal policy that orders widgets for keyboard traversal
+  /// based on the widget hierarchy order.
```

Comment:
You'll need to describe the argument here too.  You might want to make a template with `{@template ...}` and `{@macro ...}` (see examples elsewhere in the framework).